components/influx/influx.cpp

```diff
@@ -399,6 +399,13 @@ void Influx::write()
              m_stats.pwr_vout, m_stats.pwr_iout, m_stats.pwr_pout, m_stats.total_blocks_found,
              m_stats.duplicate_hashes);
 
+    // Append RTT value only if it's valid (> 0); avoids sending meaningless 0.0 on startup
+    if (m_stats.last_ping_rtt > 0.0f) {
```

Comment:
I wouldn't make it optional, if it's 0, it's 0. Would just append it above

Reply:
because missing values in the influxdb for a query can be sometimes lead to weird behaviour